### ISSUE TITLE
P#48307 Remove unsupported data types - Data types still visible in some lists

### ISSUE THREAD

**Issue by michaelarnold-dev** [onOfficeGmbH/oo-wp-plugin#151]
Original remark: For me with version 2.14.1-8-g0326ab32 installed, the data types mentioned in the requirements are all still displayed for the address list, property list and unit list. Everything is OK with all other lists and forms, the data types have all been removed.

Requirement: Separator line, black hint, red hint, file and user are no longer selectable in the configuration for any list or form in the backend.

=> For forms the error is fixed, but I can confirm that the fields are still shown for address and estate views (and probably unit lists)

https://user-images.githubusercontent.com/77264788/145575848-60d8629f-2d4d-4a28-b7c5-e57ef1730871.mp4

![Data_types_propery_list](https://user-images.githubusercontent.com/77264788/145575859-1e94d1ea-3e97-492e-a79d-785840f22b0b.png)
.

**Comment by tung-le-esg** [comment]
I will check it

**Comment by nglelinh** [comment]
@michaelarnold-dev the data type returns from onOffice API is now as TEXT, which is not correct. Please check again with the API team

![image](https://user-images.githubusercontent.com/5602479/148485292-340adfdb-8636-4ed3-943e-33c4b34c5e74.png)

**Comment by nglelinh** [comment]
![api_return_type_data](https://user-images.githubusercontent.com/5602479/148511270-dc40a36a-eb37-4334-8a52-4db3f30c3f9e.png)
![api_return_type_fail](https://user-images.githubusercontent.com/5602479/148511275-745271f3-72d9-40bc-83e1-31459f4abbc5.png)
![beta_13809-edit](https://user-images.githubusercontent.com/5602479/148511279-f80bb024-c012-418a-ab7d-ec76ba1b29a1.png)
More details here please. The type should be as in the UI

**Comment by nglelinh** [comment]
and here is the request details @michaelarnold-dev 
```
object(onOffice\SDK\internal\HttpFetch)#1736 (3) {
  ["_url":"onOffice\SDK\internal\HttpFetch":private]=>
  string(42) "https://api.onoffice.de/api/latest/api.php"
  ["_postData":"onOffice\SDK\internal\HttpFetch":private]=>
  string(388) "{"token":"121d46156a2bfb183b5ef4f020b17a82","request":{"actions":[{"actionid":"urn:onoffice-de-ns:smart:2.5:smartml:action:get","identifier":null,"parameters":{"labels":true,"language":"ENG","modules":["address","estate"],"realDataTypes":true,"showContent":true,"showTable":true},"resourceid":"","resourcetype":"fields","timestamp":1641543241,"hmac":"2a170859a2839767d1b64d3365fddd5b"}]}}"
  ["_curlOptions":"onOffice\SDK\internal\HttpFetch":private]=>
  array(2) {
    [64]=>
    bool(true)
    [181]=>
    int(2)
  }
}
```

**Comment by jayay** [comment]
I removed the `token` and `hmac` values from the request

**Comment by jmaas-onoffice** [comment]
Fixed in #109.
